### ISSUE TITLE
Include quasar sources in npm packages

### ISSUE THREAD

**Issue by mlenormand** [rstoenescu/quasar-framework#19]
It would be a good thing to include all the repo files in the npm published package.

Use case : override Quasar components, for example for override the dropdown component : 

MyCustomDropdown.vue (redefines the Quasar dropdown component but inherits the same mixin : 

```
<template>

<div class="quasar-dropdown">
    <div class="quasar-dropdown-target" @click="toggle">
        <slot name="target"></slot>
    </div>
    <div v-el:menu class="quasar-dropdown-menu" style="display: none;">
        <slot></slot>
    </div>
</div>

</template>

<script>

import vm from 'quasar-framework/src/vue-components/dropdown/dropdown-common.js'
export default {
    mixins: [vm],
    destroy() {
        document.body.removeEventListener('mousedown', this.close)
    },
    methods: {
        open(event) {
            console.log('open')
        },
        mycustommethod() {

        }
    }
}

</script>

```
Several open source projects do it, including vue, vue-router, vue-i18n, etc.

**Comment by rstoenescu** [comment]
Will do. Scheduled for v0.5.1 in a few days max.

**Comment by rstoenescu** [comment]
Will be available in v0.5.1 in a few days.
